web3.js/test/connection.test.ts

```diff
@@ -3804,6 +3804,10 @@ describe('Connection', function () {
   }
 
   if (process.env.TEST_LIVE) {
+    it('getStakeMinimumDelegation', async () => {
+      const {value} = await connection.getStakeMinimumDelegation();
+      expect(value).to.eql(1000000000);
```

Comment:
This test isn't testing the `JavaScript` implementation, it's testing the setting of the _actual_ minimum delegation, which can change over time. A change in the value of the minimum delegation should not cause the _JavaScript_ tests to break, since there would be nothing wrong with the JS implementation in that case – it's just reporting what it's told.

If you still want to write a live test for this, maybe assert _that_ it's a number, but not exactly which number.